addons/project/static/src/views/project_task_kanban/project_task_kanban_compiler.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module **/
+
+import { KanbanCompiler } from "@web/views/kanban/kanban_compiler";
+import { append, createElement } from "@web/core/utils/xml";
+
+export class ProjectTaskKanbanCompiler extends KanbanCompiler {
+    setup() {
+        super.setup();
+        this.compilers.push(
+            { selector: ".subtask_list_button", fn: this.compileSListButton },
+            { selector: "div.kanban_bottom_subtasks_section", fn: this.compileSListComponent },
+        );
+    }
+
+    /**
+     * @param {Element} el
+     * @returns {Element}
+     */
+    compileSListButton(el) {
+        el.setAttribute("t-on-click", `() => state.folded = !state.folded`);
+        el.removeAttribute("class");
+        el.setAttribute("t-attf-class", "subtask_list_button btn fa {{ state.folded ? 'fa-solid fa-angle-right' : 'fa-solid fa-angle-down' }}");
```

Comment:
```suggestion
        el.setAttribute("t-attf-class", "${el.classNames} btn fa {{ state.folded ? 'fa-solid fa-angle-right' : 'fa-solid fa-angle-down' }}");
        el.removeAttribute("class");
```
I think we could also do that. 🙂 

(not mandatory change it is up to you)

Reply:
here, I am running into errors because when the ```el.classNames``` is evaluated, ```el``` is not recognised as the function's parameter, thus I left it as is